src/moving_platform.hpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file moving_platform.hpp
+ * @brief     Moving platform class
+ * @author    Olivier verwoerd & Maarten Wassenaar
+ * @license   See LICENSE
+ */
+
+#ifndef MOVING_PLATFORM_HPP
+#define MOVING_PLATFORM_HPP
+
+namespace MovingPlatform {
+/**
+ * @brief Platform

```

Comment:
Doxygen knows which class/function this comment paragraph corresponds to and shows that name as the title, so simply copying the name is useless.
Change this and all other @brief's in this file that have just the name to a more descriptive sentence.